src/data/rbtree/init.lean

```diff
@@ -0,0 +1,218 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura
+-/
+
```

Comment:
Can you add a brief comment explaining why this is called `init`? I assume the reason is "it used to be part of the prelude"?